Library/Homebrew/rubocops/shared/desc_helper.rb

```diff
@@ -106,6 +111,7 @@ def desc_problem(message)
           correction.gsub!(/^\s+/, "")
           correction.gsub!(/\s+$/, "")
           correction.gsub!(/\.$/, "")
+          correction.gsub!(/\s?\p{So}/, "")
```

Comment:
This could leave a trailing space at the beginning, if the first character of the description is an emoji.

Easy fix for that is to move this 3 lines up.

Reply:
Done! Thank you!